lib/theme.ts

```diff
@@ -0,0 +1,74 @@
+export function theme(focusBgColor: string, backgroundColor: string) {
+  return {
+    root: {
+      primaryColor: "rgb(8 117 225)",
+      dangerColor: "salmon",
+      warningColor: "yellow",
+    },
+    sidebar: {
+      // TODO: White Logo needs to be added on the config side, then the logo can be removed from here
+      logo: `http://localhost:3000/images/white-logo.png`,
```

Comment:
I can access the image from the url, but the image wasnt pulling during space creation

Reply:
The space creation is happening in Flatfile and it can't hit your localhost. Would have to use ngrok to test